TanukiKit.xcodeproj/project.pbxproj

```diff
@@ -11,9 +11,9 @@
 		233A0B0E1C0EF8E900072A58 /* TanukiKit.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 233A0B031C0EF8E900072A58 /* TanukiKit.framework */; };
 		233A0B211C0EF94400072A58 /* Configuration.swift in Sources */ = {isa = PBXBuildFile; fileRef = 233A0B1D1C0EF94400072A58 /* Configuration.swift */; };
 		233A0B221C0EF94400072A58 /* TanukiKit.swift in Sources */ = {isa = PBXBuildFile; fileRef = 233A0B1E1C0EF94400072A58 /* TanukiKit.swift */; };
-		233A0B231C0EF94400072A58 /* Repositories.swift in Sources */ = {isa = PBXBuildFile; fileRef = 233A0B1F1C0EF94400072A58 /* Repositories.swift */; };
+		233A0B231C0EF94400072A58 /* Projects.swift in Sources */ = {isa = PBXBuildFile; fileRef = 233A0B1F1C0EF94400072A58 /* Projects.swift */; };
```

Comment:
Quick question, I also changed the func names to `projects` instead of `repos`, etc. For this one, would you like me to also give repo names, so `visibleRepos` instead of `visibleProjects` and change the documentation to `fetches repos` or is changing filename enough?